conanfile.py

```diff
@@ -124,8 +125,9 @@ class CelixConan(ConanFile):
         "build_launcher": False,
         "build_promises": False,
         "build_pushstreams": False,
-        "celix_cxx14": False,
-        "celix_cxx17": False,
+        "build_experimental": False,
```

Comment:
IIRC, device_access has been removed. Assuming no user exists, we should also remove the corresponding option.